index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New-Year-party</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+  <body>
+    <section class="main-container">
```

Comment:
- [ ] I recommend setting `margin: 0` to `body`. Because currently there is whitespace of 8px around the content:
<img width="783" alt="Screenshot 2022-04-14 at 16 28 40" src="https://user-images.githubusercontent.com/77038610/163390632-504f529c-0b4a-450f-af55-c6fc108c9260.png">